scripts/changelog.js

```diff
@@ -106,7 +106,7 @@ Please refer to the commit list given above for a complete list of changes, or w
 As with all pre-releases, this version requires explicit use of the \`next\` tag to be installed:
 
 \`\`\`sh
-npm install -g thelounge@next
+[sudo] yarn global add thelounge@next
```

Comment:
Remove sudo, you don't need root for yarn global, and we shouldn't recommend that.

Reply:
should be changed in readme here too then https://github.com/thelounge/thelounge/blob/master/README.md#running-stable-releases-using-yarn-recommended